frontend/src/global_styles/layout/work_packages/_table.sass

```diff
@@ -54,7 +54,7 @@
   > .toolbar-container
     margin-top: 5px
 
-  input[type="text"].toolbar--editable-toolbar
+  .toolbar--editable-toolbar
```

Comment:
This is a change to make sure the input-shim takes all the style overrides that the input itself has. However, for that to work, the selectors have to target a bit broader than they are currently doing. This decreases selector specificity, and could thus break something, but it looks like everything is fine.

All in all, decreased selector specificity is something that is positive anyway.